src/bin/rbw-agent/agent.rs

```diff
@@ -73,11 +78,32 @@ impl Agent {
         self,
         listener: tokio::net::UnixListener,
     ) -> anyhow::Result<()> {
+        let _ = crate::actions::subscribe_to_notifications(self.state.clone()).await;
```

Comment:
same here

Reply:
Fixed in [d6bbc46](https://github.com/doy/rbw/pull/115/commits/d6bbc46089b519eb82ae009bd48fa6d449f07941)